src/Esfa.Vacancy.Register.UnitTests/Esfa.Vacancy.Register.UnitTests.csproj

```diff
@@ -105,20 +105,20 @@
     <Reference Include="System.Xml.Linq" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="Api\App_Start\GivenAnApiFilterConfig.cs" />
-    <Compile Include="Api\App_Start\GivenVacancyApiExceptionHandler.cs" />
-    <Compile Include="Api\Mappings\IntToEnumConverterTests.cs" />
+    <Compile Include="Shared\Api\App_Start\GivenAnApiFilterConfig.cs" />
```

Comment:
You've gone and done it now haven't ya 🗡

Reply:
👍